src/etc/inc/gwlb.inc

```diff
@@ -5,6 +5,7 @@
     Copyright (C) 2008 Bill Marquette <bill.marquette@gmail.com>
     Copyright (C) 2008 Seth Mos <seth.mos@dds.nl>
     Copyright (C) 2010 Ermal Luçi
+    Copyright (C) 2018 Christoph Engelbert <me@noctarius.com>
```

Comment:
I don't agree with this one as a means of substantial copyrightable changes. interfaces.inc and interfaces.php are ok though because I expected that copyright to merely shift from dslite.inc to interfaces.inc when asking for the merge and noting now the copyright was missing.